mev_inspect/inspect_block.py

```diff
@@ -91,6 +93,12 @@ async def inspect_block(
     delete_liquidations_for_block(inspect_db_session, block_number)
     write_liquidations(inspect_db_session, liquidations)
 
+    punk_bids = get_punk_bids(classified_traces)
+    punk_bid_acceptances = get_punk_bid_acceptances(classified_traces)
+
+    punk_snipes = get_punk_snipes(punk_bids, punk_bid_acceptances)
+    logger.info(f"Block: {block_number} -- Found {len(punk_snipes)} punk snipes")
```

Comment:
Why is only punk_snipes logged? When len(punk_snipes) == 0 but len(punk_bid_acceptances) > 0, would it not still be useful to still know if any punk transfers occurred in the block?

Reply:
I was thinking that punk_snipes were the only ones that were MEV relevant. There will probably be many blocks with punk bids but no punk snipes.